conmon/cmsg.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2016 SUSE LLC
```

Comment:
@cyphar I'm not arguing with common practice, but someone (a Debian contributor, maintainer, or developer) needs to collate all those copyright statements, so it's manual work for somebody; for example: http://metadata.ftp-master.debian.org/changelogs/main/s/systemd/systemd_232-8_copyright